.gitignore

```diff
@@ -4,6 +4,10 @@
 # or operating system, you probably want to add a global ignore instead:
 #   git config --global core.excludesfile '~/.gitignore_global'
 
+# Ignore yarn bin as prescribed by scalingo at
+# https://doc.scalingo.com/languages/ruby/rails/start#specifying-a-custom-nodejs-runtime
+/bin/yarn
```

Comment:
As recommended by the scalingo documentation, I removed yarn binary and ignore it for everyone not using docker can reinstall it.